<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/yandex_music.png</normaloff>Icons/yandex_music.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 63, 63);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
color:#fff;</string>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>581</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color: rgb(84, 84, 84);
border-radius:20px;color:#fff}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>198</y>
       <width>210</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light Condensed</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{color:#fff;
background-color:rgba(0,0,0,0);}

</string>
     </property>
     <property name="text">
      <string>Кастомный путь</string>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="opendownloadbtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>253</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{color:#fff;
background-color:rgba(0,0,0,0);
border: 2px solid #fff;
border-radius:10px;}
QPushButton:hover{background-color:rgba(230,230,230,100)}
QPushButton:pressed{background-color:rgba(255,255,255,150)}
QPushButton:disabled{background-color: rgb(45, 45, 45);
border:None}
</string>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pathdownload">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>254</y>
       <width>350</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light Condensed</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{border:None;
background-color: rgba(255, 255, 255, 0);
border-bottom:2px solid rgb(236, 236, 236);
color:#fff;
border-radius:5px;}



</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>230</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light Condensed</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Папка для скачивания треков:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>290</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light Condensed</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Папка для иконок:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openiconbtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>313</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{color:#fff;
background-color:rgba(0,0,0,0);
border: 2px solid #fff;
border-radius:10px;}
QPushButton:hover{background-color:rgba(230,230,230,100)}
QPushButton:pressed{background-color:rgba(255,255,255,150)}
QPushButton:disabled{background-color: rgb(45, 45, 45);
border:None}
</string>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pathicon">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>314</y>
       <width>350</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light Condensed</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:None;
background-color: rgba(255, 255, 255, 0);
border-bottom:2px solid rgb(236, 236, 236);
color:#fff;
border-radius:5px;

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="login">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>350</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light Condensed</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{border:None;
background-color: rgba(255, 255, 255, 0);
border-bottom:2px solid rgb(236, 236, 236);
color:#fff;
border-radius:5px;}



</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>481</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Condensed</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Имя пользователя/вы не залогинины</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>350</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light Condensed</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{border:None;
background-color: rgba(255, 255, 255, 0);
border-bottom:2px solid rgb(236, 236, 236);
color:#fff;
border-radius:5px;}



</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginbtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{color:#fff;
background-color:rgba(0,0,0,0);
border: 2px solid #fff;
border-radius:10px;}
QPushButton:hover{background-color:rgba(230,230,230,100)}
QPushButton:pressed{background-color:rgba(255,255,255,150)}
QPushButton:disabled{background-color: rgb(45, 45, 45);
border:None}
</string>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutbtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{color:#fff;
background-color:rgba(0,0,0,0);
border: 2px solid #fff;
border-radius:10px;}
QPushButton:hover{background-color:rgba(230,230,230,100)}
QPushButton:pressed{background-color:rgba(255,255,255,150)}
QPushButton:disabled{background-color: rgb(45, 45, 45);
border:None}
</string>
     </property>
     <property name="text">
      <string>Выйти</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
